src/main/java/dto/quotes.java

```diff
@@ -13,6 +13,6 @@ class quotes {
     private double bidPrice;
     private int bidSize;
     private String timestamp;
-    private String condition[];
```

Comment:
C-Style array declaration. Shifted it to Java style.

Reply:
https://stackoverflow.com/questions/129178/difference-between-int-array-and-int-array

Both of these are fundamentally the same, but in a Java group project, it's wise to stick with traditional Java declarations to avoid confusion.

Reply:
Hun.  Interesting.